src/main/kotlin/org/partiql/scribe/shell/ShellHighlighter.kt

```diff
@@ -13,18 +13,18 @@
  */
 package org.partiql.scribe.shell
 
-import org.antlr.v4.runtime.BaseErrorListener
-import org.antlr.v4.runtime.CharStreams
-import org.antlr.v4.runtime.CommonTokenStream
-import org.antlr.v4.runtime.RecognitionException
-import org.antlr.v4.runtime.Recognizer
 import org.jline.reader.Highlighter
 import org.jline.reader.LineReader
 import org.jline.utils.AttributedString
 import org.jline.utils.AttributedStringBuilder
 import org.jline.utils.AttributedStyle
 import org.partiql.parser.antlr.PartiQLParser
 import org.partiql.parser.antlr.PartiQLTokens
+import org.partiql.parser.thirdparty.antlr.v4.runtime.BaseErrorListener
```

Comment:
Changes in this file are due to upgrading to PLK 0.14.6, which internalized the ANTLR dependency. ANTLR runtime dependencies are available under `org.partiql.parser.thirdparty.antlr.v4.runtime` rather than `org.antlr.v4.runtime`.